lib/autoyast.rb

```diff
@@ -262,26 +262,25 @@ def apply_url_extraction(xml)
     end
   end
 
-  def apply_extracted_files(scope)
-    return if !@system_description[scope] || !@system_description[scope].extracted
+  def apply_changed_files(scope)
```

Comment:
Assignment Branch Condition size for apply_changed_files is too high. [51.25/15]